src/logger.coffee

```diff
@@ -1,32 +1,33 @@
-winston = require 'winston'
 
-config =
+winston = require('winston')
+
+
+module.exports = new (winston.Logger)({
+  transports: [
+    new (winston.transports.Console)({colorize: true})
+  ]
   levels:
-    silly: 0
-    debug: 1
-    verbose: 2
-    info: 3
-    test: 4
-    pass: 5
-    fail: 6
+    silly: 14
```

Comment:
They're just reversed and then I sorted them. It's change in the Winston logger package.